app.py

```diff
@@ -1,25 +1,159 @@
-from flask import Flask, render_template
-import db_utils as db_utils
+# This will be for flask
+import flask
+from flask import Flask, render_template, request
+from flask import jsonify
+from scraper import scraper
+from main import Main
+from flask_cors import CORS, cross_origin
 
-app = Flask(__name__)
+app = flask.Flask(__name__)
 
-@app.route('/')
+app.add_url_rule('/',
+                 view_func=Main.as_view('main'),
+                 methods=["GET"])
+app.add_url_rule('/<page>/',
+                 view_func=Main.as_view('page'),
+                 methods=["GET"])
 
-@app.route('/stations')
+     
+@app.errorhandler(404)
+@cross_origin()
+def page_not_found(error):
+    return flask.render_template('404.html'), 404
+
+@app.route("/stations")
+@cross_origin()
 def get_stations():
-    db_utils.get_stations()
+    engine = scraper.connect_db("ubuntu@ec2-18-206-204-211.compute-1.amazonaws.com:22", "3306", " dublin-bikes", "admin", "0d72d379765757bc1")
+    sql = "select * from bike_stations;"
+    rows = engine.execute(sql).fetchall()
+    print("#found {} stations", len(rows))
+    stations = jsonify(stations=[dict(row) for row in rows])    
+    engine.dispose()
+    return stations
```

Comment:
this work looks great but I think I already did all of this in `db_utils` - was there a reason why you chose to remove these? Example:

```
def get_stations():
    db_utils.get_stations()
```

Maybe it was just a mistake, which is okay!